dotCMS/src/integration-test/java/com/dotcms/util/MimeTypeUtilsTest.java

```diff
@@ -0,0 +1,65 @@
+package com.dotcms.util;
+
+import com.dotcms.UnitTestBase;
+import com.dotcms.repackage.org.apache.commons.io.FileUtils;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.Date;
+
+/**
+ * MimeTypeUtils unit test.
+ * @author jsanca
+ */
+public class MimeTypeUtilsTest extends UnitTestBase {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4801734)